.hygen/generate/relational-resource/__mock__/mock.ejs.t

```diff
@@ -1,24 +1,33 @@
 ---
-to: src/<%= h.inflection.transform(name, ['pluralize', 'underscore', 'dasherize']) %>/__mock__/<%= h.inflection.transform(name, ['underscore', 'dasherize']) %>.mock.ts
+to: "<%= isAddTestCase ? `src/${h.inflection.transform(name, ['pluralize', 'underscore', 'dasherize'])}/__mock__/${h.inflection.transform(name, ['underscore', 'dasherize'])}.mock.ts` : null %>"
 ---
+<% if (functionalities.includes('findAll')) { %>
 import { IPaginationOptions } from "@src/utils/types/pagination-options";
+<% } %>
+<% if (functionalities.includes('create')) { %>
 import { Create<%= name %>Dto } from '@src/<%= h.inflection.transform(name, ['pluralize', 'underscore', 'dasherize']) %>/dto/create-<%= h.inflection.transform(name, ['underscore', 'dasherize']) %>.dto';
+<% } %>
+<% if (functionalities.includes('update')) { %>
 import { Update<%= name %>Dto } from '@src/<%= h.inflection.transform(name, ['pluralize', 'underscore', 'dasherize']) %>/dto/update-<%= h.inflection.transform(name, ['underscore', 'dasherize']) %>.dto';
+<% } %>
 import { <%= name %> } from '@src/<%= h.inflection.transform(name, ['pluralize', 'underscore', 'dasherize']) %>/domain/<%= h.inflection.transform(name, ['underscore', 'dasherize']) %>';
 // __mock__/<%= h.inflection.transform(name, ['underscore', 'dasherize']) %>.mock.ts
+<% if (functionalities.includes('findAll')) { %>
```

Comment:
combine the conditional imports of findAll 
**line no 15
line no 4**

Reply:
I think it's fine as line 4 is an import statement and 15 is actual data